src/modules/Packages.rb

```diff
@@ -1555,12 +1553,10 @@ def AdjustSourcePropertiesAccordingToProduct(src_id)
       new_name = nil
       Builtins.foreach(all_products) do |one_product|
         # source ID matches
-        if one_product.source == src_id
-          if one_product.name != ""
-            new_name = one_product.name
-            Builtins.y2milestone("Product name found: %1", new_name)
-            raise Break
-          end
+        if one_product.source == src_id && (one_product.name != "")
```

Comment:
Are the parenthesis needed?

Reply:
probably not. Let me try what rubocop says, but it is in general now more strict about operator precedence and prefers parentheses to avoid confusion